legion/legion/utils.py

```diff
@@ -203,17 +203,26 @@ class Colors:
     UNDERLINE = '\033[4m'
 
 
-def normalize_name(name):
+def normalize_name(name, dns_1035=False):
```

Comment:
This should be in k8s

Reply:
But models use this functional to normalize their names

Reply:
Have come to the conclusion that this is left here